docs/source/guide/ses-filters.rst

```diff
@@ -0,0 +1,105 @@
+.. Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+   This work is licensed under a Creative Commons Attribution-NonCommercial-ShareAlike 4.0
+   International License (the "License"). You may not use this file except in compliance with the
+   License. A copy of the License is located at http://creativecommons.org/licenses/by-nc-sa/4.0/.
+
+   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+   either express or implied. See the License for the specific language governing permissions and
+   limitations under the License.
+   
+.. _aws-boto3-ses-filters:  
+
+###################################
+Managing Email Filters with SES API 
+###################################
+
+.. meta::
+   :description: Use the Amazon SES API to manage email filters.
+   :keywords: Amazon SES code examples for Python, IP Address Email Filters with Python
+
+In addition to sending emails, you can also receive email with Amazon Simple Email Service (SES). An IP address filter enables you to optionally specify whether to accept or reject mail that originates from an IP address or range of IP addresses. For more information, see `Managing IP Address Filters for Amazon SES Email Receiving <http://aws.amazon.com/documentation/ses/receiving-email-managing-ip-filters.html.html>`__.
+
+The following examples show how to:
+
+* Create an email filter using `create_receipt_filter() <https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ses.html#SES.Client.create_receipt_filter.html>`__.
+* List all email filters using `list_receipt_filters() <https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ses.html#SES.Client.list_receipt_filters.html>`__.
+* Remove an email filter using `delete_receipt_filter() <ahttps://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ses.html#SES.Client.delete_receipt_filter.html>`__.
+
+Prerequisite Tasks
+==================
+
+To set up and run this example, you must first complete these tasks:
+
+* Configure your AWS credentials, as described in :doc:`quickstart`.
+
+Create an Email Filter
+======================
+
+To allow or block emails from a specific IP address, use the `CreateReceiptFilter <https://docs.aws.amazon.com/ses/latest/APIReference/API_CreateReceiptFilter.html.html>`__ operation. Provide the IP address or range of addresses and a unique name to identify this filter.
+
+Example
+-------
+
+.. code-block:: python
+
+    import boto3
+
+    # Create SES client
+    ses = boto3.client('ses')
+
+    # Create receipt filter
+    response = ses.create_receipt_filter(
+      Filter = {
+        'NAME'     : 'NAME',
+        'IpFilter' : {
+          'Cidr'   : 'IP_ADDRESS_OR_RANGE',
+          'Policy' : 'Allow' | 'Block'
```

Comment:
Can we update this so that its valid Python? Its a little strange to have our sample code not be valid python.